lldb/source/Target/InstrumentationRuntime.cpp

```diff
@@ -60,6 +60,8 @@ void InstrumentationRuntime::ModulesDidLoad(
       if (CheckIfRuntimeIsValid(module_sp)) {
         SetRuntimeModuleSP(module_sp);
         Activate();
+        if (IsActive())
+          SetRuntimeModuleSP({}); // Don't cache module if activation failed.
```

Comment:
Should this be `if (!IsActive())`?

Reply:
Thanks Jonas. Yes, this should have been amended.